_drafts/2019-01-21-Increasing-the-test-coverage.md

```diff
@@ -10,46 +10,29 @@ Refactoring code with poor test coverage is always difficult.
 The front end apps have a few tests in place but by far not enough,
 so we'll need to do something about it.
 
+We talked about this on the 17th Jan 2019 on in the #team-apps Slack channel
+and this is a summary on what we agreed.
+
 ## Add tests
 
-If you've followed the discussion in the team-apps chat on the 17th Jan 2019,
-you'll see the we agreed on the following things:
+We agreed on the following things:
 
-* Each PR should contain a few test
+* Each PR should contain a few tests when appropriate
 * React (or other view) components don't need to be tested
   * This can be done by e2e tests, with cypress or should be done in component libraries
 
-Ideally you'll test new code 100% and add tests for bugs that you're fixing.<br>
-Also adding a few tests when you encounter old code can be of use, but like
-I said, you don't need to overdo it.
+Ideally you'll add tests that make sense, don't test everything just for the sake
+of having everything tested, it'll make the code resilient to change 
```

Comment:
Is it possible that you mean "resistant" instead of "resiliant"?

Reply:
Thanks, fixed.